juniper_codegen/src/lib.rs

```diff
@@ -503,9 +503,9 @@ pub fn derive_enum(input: TokenStream) -> TokenStream {
 /// }
 /// ```
 /// > __NOTE:__ Once we provide all 3 custom functions, there is no sense to
```

Comment:
```suggestion
/// > __NOTE:__ Once we provide all 3 custom functions, there is no sense
```

Reply:
@LegNeato "no sense in following" sounds nicer to me, than just "no sense following". Would it be okay?

Reply:
```suggestion
/// > __NOTE:__ Once we provide all 3 custom functions, there is no sense in
```